tini/src/seccomp_fd_notify.c

```diff
@@ -39,8 +43,13 @@
 		fprintf(stderr, "\n");                                         \
 	}
 
+typedef struct tsa_fds {
```

Comment:
Why a typedef and not a plain struct?

Reply:
just to avoid writing the `struct` in `struct tsa_fds`